requirements.txt

```diff
@@ -1,12 +1,11 @@
-PyYaml >= 5.1
-requests >=2
-Jinja2
-toml
-ensureconda >=1.3
 click >=8.0
 click-default-group
-pydantic >=1.8.1
+ensureconda >=1.3
+jinja2
 poetry <1.2
+pydantic >=1.8.1
+pyyaml >= 5.1
+requests >=2
 ruamel.yaml
-poetry
+toml
```

Comment:
should probably start using `tomli`...

Reply:
What's the difference?

Reply:
sorry, confused with `pytoml`, which is definitely deprecated.

https://github.com/uiri/toml hasn't had a substantive PR merged in some time, and many open issues, but is probably still fine. 

I guess the biggest thing in favor of `tomli` is it's just the parsing side, and is the basis for the upcoming `tomllib` in stdlib: https://peps.python.org/pep-0680/#rationale

Reply:
If you want to make a or to shift to tomli (or I guess preferentially the stdlib one for py 3.11/12) that would be awesome.